src/load-binding.ts

```diff
@@ -193,3 +197,5 @@ export const NativeIterator: typeof NativeIteratorCls = binding.Iterator;
 export const NativeTransaction: NativeTransaction = binding.Transaction;
 export const TransactionLog: TransactionLog = binding.TransactionLog;
 export const version: string = binding.version;
+
+process.on('exit', binding.shutdown);
```

Comment:
Hmm, I'm not a huge fan of libraries listening to `on('exit')`. I would prefer we document in the readme that users of this library should add this line to their app.